utils/src/main/kotlin/FileMatcher.kt

```diff
@@ -32,7 +30,7 @@ class FileMatcher(
      *
      * [1]: https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob
      */
-    val patterns: List<String>
+    val patterns: List<String>,
```

Comment:
Omit the trailing comma.

Reply:
That's not possible.

Reply:
What am I missing? It was not there before, so why is it needed now?

Reply:
You're right. The comma should be added in the next commit.